shared/desktop/yarn-helper/font.tsx

```diff
@@ -278,10 +278,10 @@ function updateIconConstants() {
     })
   // dark
   iconFiles
-    .filter(i => i.startsWith('iconDark-'))
+    .filter(i => i.startsWith('icon-dark-'))
     .forEach(i => {
       const shortName = i.slice(0, -4)
-      const lightName = shortName.replace(/^iconDark-/, 'icon-')
```

Comment:
i did this on purpose since this breaks ever calling a thing `dark-whatever` but its fine

Reply:
We can change it back if we need to. The problem I ran into is how Zeplin exports our assets (it converts `iconDark`, how I named them in Sketch, to `icon-dark`).